### PR TITLE
enable cluster access to initiators

### PR DESCRIPTION
**Pull request by hoxhaeris**

### CODE REVIEW

**Comment by kaovilai** [comment]
this is awesome

**Comment by hoxhaeris** [comment]
/hold
2 new Cluster-Bot App OAuth scopes should be added before merging:
- users:profile:read
- users:read:email

File: manager.go
Comment (osherdp) [inline manager.go:945]: should it be ``Debugf`` maybe? I'm not entirely familiar with the logging scheme for this component but it seems more appropriate to write external-dependent/non-critical logs with the debug level

File: prow.go
Comment (osherdp) [inline prow.go:872]: maybe it should be split to two conditions (for checking existence of the namespace and creation of rbac)?
that's because rbac creation might be because of other reasons

**Review by osherdp — COMMENTED** [review]
looks great!
just added a couple of comments

File: prow.go
Comment (bradmwilliams) [inline prow.go:869]: It's probably best if we dont start spraying folks RedHat ID's throughout our logs.  I'd recommend only printing their unique slack ID (`job.RequestedBy`).

File: prow.go
Comment (bradmwilliams) [inline prow.go:874]: It's probably best if we dont start spraying folks RedHat ID's throughout our logs. I'd recommend only printing their unique slack ID (`job.RequestedBy`).

File: prow.go
Comment (bradmwilliams) [reply prow.go:869]: This particular error should probably be logged as an Errorf()

File: prow.go
Comment (bradmwilliams) [inline prow.go:880]: This should probably be an Errorf() as well

File: prow.go
Comment (hoxhaeris) [reply prow.go]: Fixed

File: prow.go
Comment (hoxhaeris) [reply prow.go]: Fixed

File: prow.go
Comment (hoxhaeris) [reply prow.go]: Fixed

File: manager.go
Comment (hoxhaeris) [reply manager.go:945]: This was changed to `Errorf` instead.

File: prow.go
Comment (hoxhaeris) [reply prow.go:875]: Generally, we want to retry on any type of failure.
The namespace one is noted since it occurs quite frequently (due to a race condition).

**Review by bradmwilliams — APPROVED** [review]

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**
